### PR TITLE
Fix: uninitialized constant FactoryBot::Syntax on Heroku

### PR DESCRIPTION
**Pull request by newstler**

### CODE REVIEW

**Comment by iwan** [comment]
Got the same error on Render

**Comment by newstler** [comment]
> 

On this branch?

**Comment by olbrich** [comment]
This probably shouldn't even be in an initializer, but should be done in `test_helper.rb`. If there's a good reason for this elsewhere it should be documented.

File: test/test_helper.rb [hunk 1]
```diff
@@ -27,3 +27,9 @@
 
   # Add more helper methods to be used by all tests here...
 end
+
+if defined?(FactoryBot)
```
Comment (olbrich) [inline test/test_helper.rb:31]: I don't think you need the conditional anymore since FactoryBot is a dependency in the test group.

**Comment by newstler** [comment]
@olbrich Good point! Moved code to `test_helper.rb` and updated PR.

**Pull request merged**